docs/build-a-runtime/index.md

```diff
@@ -89,12 +89,10 @@ The above diagram details the two high-level components of the base9 architectur
 Our frontend language is a primitive subset of JavaScript. Currently, it can only operate on integers. Let's have a look at some code:
```

Comment:
Lol we still need to fix this, feel like tacking it on?

Reply:
haha yeah... I have it fixed in a different branch somewhere, which basically rewrites the intro. I'm trying to avoid big changes to the docs until that goes in. I can probably get to it later today though.

Reply:
Ahh I see